src/browser/modules/Stream/Auth/ConnectForm.tsx

```diff
@@ -0,0 +1,235 @@
+    if (!selection) {
+      return
+    }
+    e.preventDefault()
+    e.stopPropagation()
+    let val = selection.toString()
+    if (scheme) {
+      val = `${scheme}${stripScheme(val)}`
+    }
+    e.clipboardData?.setData('text', val)
+  }
+
+  const onHostChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const val = e.target.value
+    props.onHostChange(getScheme(scheme), val)
+  }
+
+  const onSchemeChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    const val = e.target.value
+    props.onHostChange(getScheme(val), stripScheme(props.host))
+  }
+
+  const onConnectClick = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    setConnecting(true)
+    props.onConnectClick(() => setConnecting(false))
+  }
+
+  return (
+    <StyledConnectionForm onSubmit={onConnectClick}>
+      <FlexSpaceBetween>
+        <StyledConnectionFormEntry>
+          <StyledConnectionLabel
+            htmlFor="url-input"
+            title="Pick neo4j:// for a routed connection, bolt:// for a direct connection to a DBMS instance."
+          >
+            Connect URL
+          </StyledConnectionLabel>
+          {props.allowedSchemes && props.allowedSchemes.length ? (
+            <>
+              <StyledSegment>
+                <StyledConnectionSelect
+                  onChange={onSchemeChange}
+                  value={scheme}
+                  data-testid="bolt-scheme-select"
+                >
+                  {props.allowedSchemes.map(s => {
+                    const schemeString = `${s}://`
+                    return (
+                      <option value={schemeString} key={toKeyString(s)}>
+                        {schemeString}
+                      </option>
+                    )
+                  })}
+                </StyledConnectionSelect>
+                <StyledConnectionTextInput
+                  onCopy={onCopyBoltUrl}
+                  data-testid="boltaddress"
+                  onChange={onHostChange}
+                  value={stripScheme(props.host)}
+                  id="url-input"
+                />
+              </StyledSegment>
+              <StyledBoltUrlHintText className="url-hint-text">
+                Pick neo4j:// for a routed connection, bolt:// for a direct
+                connection to a DBMS.
+              </StyledBoltUrlHintText>
+            </>
+          ) : (
+            <StyledConnectionTextInput
+              data-testid="boltaddress"
+              onChange={onHostChange}
+              defaultValue={props.host}
+            />
+          )}
+        </StyledConnectionFormEntry>
+        {props.showDbField && (
+          <StyledConnectionFormEntry>
+            <StyledConnectionLabel>
+              Database
+              <StyledConnectionTextInput
+                data-testid="database"
+                onChange={props.onDatabaseChange}
+                value={props.database}
+                placeholder="default database"
```

Comment:
Maybe change this to "Database to use" or something

Reply:
You mean the label? I think the current placeholder makes sense

Reply:
I don't :)
It's not a default db the users enters, it's the target db that the user wants to use when a connection is made.

Reply:
Yeah and that's why the place holder is there to show what happens if you leave the field empty! Right now it think it makes sense to have a field titled "Database to use" and the default/placeholder being "default database". Having both be "database to use" I think would lead to me thinking the field was required

Reply:
I see your reasoning, but placeholders are hints on what the users should enter, not to indicate value if left blank.
The label could suggest "leave empty for default)" or something to let the users know what happens if it's left empty.